R/msnset2se.R

```diff
@@ -0,0 +1,48 @@
+#' SummarizedExperiment to MSnSet object conversion
+#'
+#' \code{se2msnset} generates an MSnSet object from an SummarizedExperiment object.
+#'
+#' @param se SummarizedExperiment,
+#' Object which will be turned into an MSnSet object.
+#' @return MSnSet object.
+#' @export
+se2msnset <- function(se) {
+  # Check input
+  assertthat::assert_that(inherits(se, "SummarizedExperiment"))
+
+  # Extract expression, feature and pheno data
+  raw <- SummarizedExperiment::assay(se)
+  feat_data <- data.frame(SummarizedExperiment::rowData(se))
+  rownames(feat_data) <- se@NAMES
+  pheno_data <- data.frame(SummarizedExperiment::colData(se))
+
+  # Generate MSnSet
+  msnset <- MSnbase::MSnSet(exprs = as.matrix(raw),
```

Comment:
There is no need to use `MSnbase::` because there are no other `MSnSet`s around and the namespace of the package is always used first.